.github/workflows/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us reproduce and fix a bug
+labels:
+  - "type: bug"
+  - "status: triage"
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Thank you for contributing! Before reporting a bug, please make sure it has no duplicate and/or has already been addressed by searching through [the existing and past issues](https://github.com/BerkeleyLearnVerify/Scenic/issues)
+
+  - type: textarea
+    attributes:
+      label: System Details
+      description: |
+        Please provide the following system information to help us diagnose the bug.
+        1. Python Version
+        2. Scenic Version
+        3. Operating System / Platform
+        4. Simulator Version (Optional, only applies if bug is simular specific)
+
+      placeholder: |
```

Comment:
One thing we might consider adding is the list of installed Python packages, i.e. the output of `pip freeze`. This might help clear up any issues with dependencies. @dfremont

Reply:
It's a good idea, but I'd only want to include it if it was minimized by default (otherwise the reports will be huge: for me a freeze produces 140 lines). If there isn't an easy way to do that, I'd say let's not worry about this -- we can always request a freeze if necessary.

Reply:
Agreed with this assessment. We can probe further information as needed!